modules/lti/src/main/resources/tools/series/series.js

```diff
@@ -115,6 +121,32 @@ function loadPage(page) {
         }
       }
 
+      // Download buttons
+      // Create a button for each track that contains a link to a mp4 or webm
+      var buttonData = [];
+      var tracks = episode.mediapackage.media.track;
+      tracks = Array.isArray(tracks) ? tracks : [tracks];
+      for (var k = 0; k < tracks.length; k++)
+      {
+        if( tracks[k].url.endsWith('mp4') || tracks[0].url.endsWith('webm')) {
```

Comment:
Spacing is inconsistent. From looking at the rest of this file, this should probably be

```js
        if (tracks[k].url.endsWith('mp4') || tracks[0].url.endsWith('webm')) {
```

Also: why only mp4 and webm? 

Also also: `tracks[0]`... is that intended or should it be `tracks[k]`? If the latter, the `for` loop should be

```js
for (const track of tracks) {
```

Reply:
Checking whether the url of a track points to a mp4 or webm file was recommended as a way to check whether the track is for a downloadable video or not.

And indeed, it should be `tracks[k]`.

Reply:
If webm and mp4 is the only thing produces by Opencast, then that's fine. I am just not sure about that.